docs/decisions/0002-text-editor-adr.md

```diff
@@ -0,0 +1,86 @@
+# Status
+
+Accepted
+
+# Context
+
+We seek to hoist the legacy text (HTML) text editor out of the monolith into a decoupled React application
```

Comment:
Note here that it is for the text xblock specifically. This is not a generalized text editor component.
```suggestion
We seek to hoist the legacy text (HTML) text xblock editor out of the monolith into a decoupled React application
```